File: Modul 06/03/PRAK603-2410817310020-M. Rizki Dinar.py
import numpy as np
n1,n2 = map(int, input().split())
arr = []
if n1 != n2:
    print("Jumlah tidak sama")
    exit()
arr[0] = list(map(int, input().split())
arr[1] = list(map(int, input().split())
arr = np.array(arr)
hasil = arr[0]*arr[1]
print(*hasil)

File: Modul 06/03/PRAK603-2410817310020-M. Rizki Dinar.c
#include <stdio.h>

int fill(int arr[], int lim){
    for(int i = 0; i<lim; i++){
        scanf("%d", &arr[i]);
    }
}

int main()
{
    int n1,n2;
    scanf("%d %d", &n1, &n2);
    int arr[2][n1];
    if(n1 != n2){
        printf("Jumlah tidak sama");
        return 0;
    }
    fill(arr[0], n1);
    fill(arr[1], n1);
    for(int i = 0; i<n1; i++){
        arr[1][i] *= arr[0][i];
        printf("%d ", arr[1][i]);
    }
    return 0;
}
